rosdep/base.yaml

```
@@ -2865,6 +2865,17 @@ libedit-dev:
   nixos: [libedit]
   rhel: [libedit-devel]
   ubuntu: [libedit-dev]
+e2fsprogs:
+  alpine: [e2fsprogs]
+  arch: [e2fsprogs]
+  debian: [e2fsprogs]
+  fedora: [e2fsprogs]
+  gentoo: [sys-libs/e2fsprogs-libs]
+  macports: [e2fsprogs]
+  nixos: [e2fsprogs]
+  opensuse: [e2fsprogs-libs]
```

This looks like the libraries, but I think you want the utilities: https://software.opensuse.org/package/e2fsprogs

```suggestion
  opensuse: [e2fsprogs]
```